pkg/api/plugin_resource_test.go

```diff
@@ -49,6 +49,13 @@ func TestCallResource(t *testing.T) {
 	require.NoError(t, err)
 
 	cfg := setting.NewCfg()
+	// TODO: These are very annoying and cause an error down the line if the sections are not present.
+	//  Imo, the tracing sections should not be required when parsing the tracing settings, so the errors
+	//  for "section not found" should not exist at all, and tracing should be disabled when they are not present.
+	cfg.Raw.Section("tracing.opentelemetry")
```

Comment:
What if we just updated the parsing logic in `pkg/infra/tracing/opentelemetry_tracing.go` to use cfg.Raw.Section for these sections instead? IE

```diff
func (ots *Opentelemetry) parseSettingsOpentelemetry() error {
-	section, err := ots.Cfg.Raw.GetSection("tracing.opentelemetry")
-	if err != nil {
-		return err
-	}
+   section := ots.Cfg.Raw.Section("tracing.opentelemetry")
```

Reply:
Good catch! I was not aware of this

Reply:
Done! :smile: